metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -160,7 +161,7 @@ def __init__(
             sink_config = self.config.sink.dict().get("config") or {}
             self.sink: Sink = sink_class.create(sink_config, self.ctx)
             logger.debug(f"Sink type:{self.config.sink.type},{sink_class} configured")
-            logger.info(f"Sink configured successfully. {self.sink.configured()}")
+            logger.debug(f"Sink configured successfully. {self.sink.configured()}")
```

Comment:
I think this is a pretty important log line. Only gets printed once in the execution of a recipe. Any particular reason to drop it to debug?